CMakeLists.txt

```diff
@@ -1,16 +1,19 @@
-# Copyright 2017 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+# Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Is it really necessary to change the copyright?

Reply:
I would take advantage of this PR to update them all (I think that they should be updated as the files are changed).